jobs/mongodb_migration/Dockerfile

```diff
@@ -5,20 +5,21 @@ FROM python:3.9.15-slim
 ENV PYTHONFAULTHANDLER=1 \
     PYTHONUNBUFFERED=1 \
     PYTHONHASHSEED=random \
-    PIP_NO_CACHE_DIR=off \
+    PIP_NO_CACHE_DIR=1 \
     PIP_DISABLE_PIP_VERSION_CHECK=on \
     PIP_DEFAULT_TIMEOUT=100 \
     POETRY_NO_INTERACTION=1 \
     # Versions:
     POETRY_VERSION=1.3.2 \
-    POETRY_VIRTUALENVS_IN_PROJECT=true
+    POETRY_VIRTUALENVS_IN_PROJECT=true \
+    PATH="$PATH:/root/.local/bin"
```

Comment:
What is the executable inside `/root/.local/bin` you are adding to PATH?

Reply:
I mean I guess there is no other executable with the same name already in PATH, because that would have precedence...

Reply:
poetry